Client/mods/deathmatch/logic/luadefs/CLuaDefs.h

```diff
@@ -26,6 +26,7 @@ class CClientPlayerManager;
 class CClientRadarMarkerManager;
 class CClientTeamManager;
 class CClientVehicleManager;
+class CClientFBX;
```

Comment:
Shouldn't this say this instead?
```suggestion
class CClientFBXManager;
```